components/public-api/gitpod/experimental/v1/scm.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package gitpod.experimental.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
+
+service SCMService {
+    // GetSuggestedRepoURLs returns a list of suggested repositories to open for
+    // the user.
+    rpc GetSuggestedRepoURLs(GetSuggestedRepoURLsRequest)
+        returns (GetSuggestedRepoURLsResponse) {}
+}
+
+message GetSuggestedRepoURLsRequest {}
+
+message GetSuggestedRepoURLsResponse { repeated string repos = 1; }
```

Comment:
While we are at it, could we enhance the response so that the list contains either repositories (maybe just a URL) but also projects, { projected: string, name:string: URL:string }? We are very soon enhancing this function to return projects as well.

Reply:
@svenefftinge in that case, would you be happy with a name like `GetSuggestedContexts` instead?

Reply:
`getSuggestedProjectsAndRepos` would be good. a context URL is more

Reply:
Actually, we should just ask getProjects from the client an merge it in. So `getSuggestedRepositoryURLs` is good.

Reply:
and then just return string again (sorry 😇 )

Reply:
All good 👍